### DISCUSSION TITLE
Public beta "Internal Server Error"

### DISCUSSION THREAD

**Discussion by AntonOfTheWoods** [fief-dev/fief#278] [Q&A]
I am trying to set up a Github test (with fastapi/docs) and after apparently successfully getting a token from GH, it redirects me to:

```
https://qman.fief.dev/oauth/callback?code=2b6e6e89a3355f4b1b25&state=oHUK0HxzOcHbWvaRQHqA8kBOnGwNFCYpkMZRzmQXRgk
```

Which just returns `Internal Server Error`. Any pointers? This also happens in a new private browsing window

**Comment by AntonOfTheWoods** [comment]
The local db and a Microsoft provider is working with on the same instance, so it looks like an issue with GH integration

**Comment by frankie567** [comment]
Have you enabled all the correct permissions and settings for your GitHub apps (see: https://docs.fief.dev/configure/oauth-providers/#github):

> * Ensure Expire user authorization tokens and Request user authorization (OAuth) during installation are checked.
> * In the User permissions list, make sure Email addresses has Read-only access.

**Reply by AntonOfTheWoods** [reply]
Whether or not I have, I'm not sure that should be causing an undocumented 500 error...
